sdks/python/setup.py

```diff
@@ -159,15 +159,14 @@ def generate_protos_first():
     # Pyproject toml build happens in isolated environemnts. In those envs,
     # gen_protos is unable to get imported. so we run a subprocess call.
     cwd = os.path.abspath(os.path.dirname(__file__))
-    p = subprocess.call([
+    p = subprocess.run([
       sys.executable,
       os.path.join(cwd, 'gen_protos.py'),
       '--no-force'
-    ])
-    if p != 0:
-      raise RuntimeError()
-  except RuntimeError:
-    logging.warning('Could not import gen_protos, skipping proto generation.')
+    ], capture_output=True, check=True)
```

Comment:
note that `capture_output=True,` will hide all of stdout, and in case of error, only stderr is printed.  is this intentional?

Reply:
printing output now